opm/autodiff/WellStateFullyImplicitBlackoil.hpp

```diff
@@ -20,7 +20,7 @@
 #ifndef OPM_WELLSTATEFULLYIMPLICITBLACKOIL_HEADER_INCLUDED
 #define OPM_WELLSTATEFULLYIMPLICITBLACKOIL_HEADER_INCLUDED
 
-
+#include <opm/autodiff/BlackoilModelEnums.hpp>
```

Comment:
The new resize is to make the WellState classes have the same resize method signature. That is needed upon restart file read.